CleanSlate/events/wol_business_events.txt

```diff
@@ -1062,13 +1421,13 @@ character_event = {
 # Steward conclusion
 character_event = {
 	id = WoL.10127
+
 	is_triggered_only = yes
-	hide_window = yes ###fix if time
+	hide_window = yes # TODO: Fix if time
```

Comment:
What needs fixed, exactly?

Reply:
I'm assuming the developer intended to add localisation and make the event visible, as the steward could be a player. Looking at it again, if that's not something we're adding in, we could just remove WoL.10127 entirely.